test/v2/getStore.test.ts

```diff
@@ -56,11 +54,12 @@ apiSuite('Component: GET /api/v2/store', (appState: AppState) => {
       })
       .expect(res => isSuccessfulResponse(res))
 
-    oldestTs = repoTimestamp = res.body.hash
+    // 4:10
+    checkpointsAtBeginning = clientCheckpoints = res.body.hash
```

Comment:
You and I know these are checkpoints, but the API still calls it the "hask". Maybe we keep the term "hash" in the tests, just for consistency with the API?